### PR TITLE
using find packages instead of hard-coding the submodules.

### PR DESCRIPTION
**Pull request by michaelaye**
the way it was before, my system could not find the labels submodule. like this all works for me.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 try:
-    from setuptools import setup
+    from setuptools import setup, find_packages
 except ImportError:
```
Comment (wtolson) [inline setup.py]: No need for this try/except block now since distutils doesn't support an equivalent find_packages.

File: setup.py [hunk 2]
```diff
@@ -19,13 +19,14 @@
...
     author_email='trevor@heytrevor.com',
     url='https://github.com/wtolson/pysis',
-    packages=[
-        'pysis',
-        'pysis.binning',
-        'pysis.util',
-    ],
-    package_dir={'pysis':
-                 'pysis'},
+    # packages=[
```
Comment (wtolson) [inline setup.py]: Could you remove these commented out lines?

**Comment by wtolson** [comment]
Ah! Glad you were able to figure out what was going on. I left a couple comments on some cleanup and then I'll merge and publish a new version to pypi. Also feel free to add yourself to [AUTHORS.rst](https://github.com/wtolson/pysis/blob/master/AUTHORS.rst)!

File: setup.py (hunk 1 above)
Comment (michaelaye) [inline setup.py]: So, are you willing to give up on distutils compatibility?

**Comment by michaelaye** [comment]
Should I bump the minor revision, if u want to push it to pypi?

File: setup.py (hunk 1 above)
Comment (wtolson) [inline setup.py]: Yeah. Its not very often used anymore. I figure we can add it back when someone needs it.

**Comment by wtolson** [comment]
I can bump the version. There is another line I wanted to add to the history.

**Comment by michaelaye** [comment]
ok, then it's done i think.

**Comment by wtolson** [comment]
Looks good thanks!

**Comment by wtolson** [comment]
Just pushed to pypi. Does this fix the issue in #7?

**Pull request merged**